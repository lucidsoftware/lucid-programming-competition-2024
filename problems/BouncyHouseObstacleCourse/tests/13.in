375
0
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99
100
101
102
103
104
105
106
107
108
109
110
111
112
113
114
115
116
117
118
119
120
121
122
123
124
125
126
127
128
129
130
131
132
133
134
135
136
137
138
139
140
141
142
143
144
145
146
147
148
149
150
151
152
153
154
155
156
157
158
159
160
161
162
163
164
165
166
167
168
169
170
171
172
173
174
175
176
177
178
179
180
181
182
183
184
185
186
187
188
189
190
191
192
193
194
195
196
197
198
199
200
201
202
203
204
205
206
207
208
209
210
211
212
213
214
215
216
217
218
219
220
221
222
223
224
225
226
227
228
229
230
231
232
233
234
235
236
237
238
239
240
241
242
243
244
245
246
247
248
249
250
251
252
253
254
255
256
257
258
259
260
261
262
263
264
265
266
267
268
269
270
271
272
273
274
275
276
277
278
279
280
281
282
283
284
285
286
287
288
289
290
291
292
293
294
295
296
297
298
299
300
301
302
303
304
305
306
307
308
309
310
311
312
313
314
315
316
317
318
319
320
321
322
323
324
325
326
327
328
329
330
331
332
333
334
335
336
337
338
339
340
341
342
343
344
345
346
347
348
349
350
351
352
353
354
355
356
357
358
359
360
361
362
363
364
365
366
367
368
369
370
371
372
373
374
835
0 1 15
1 2 18
2 3 14
3 4 18
4 5 5
5 6 23
6 7 10
7 8 12
8 9 4
9 10 25
10 11 13
11 12 27
12 13 3
13 14 19
14 15 10
15 16 16
16 17 32
17 18 17
18 19 16
19 20 4
20 21 33
21 22 8
22 23 35
23 24 17
24 25 5
25 26 13
26 27 6
27 28 4
28 29 33
29 30 24
30 31 32
31 32 29
32 33 44
33 34 27
34 35 33
35 36 5
36 37 49
37 38 31
38 39 9
39 40 48
40 41 2
41 42 24
42 43 6
43 44 33
44 45 23
45 46 8
46 47 16
47 48 44
48 49 46
49 50 41
50 51 7
51 52 8
52 53 19
53 54 36
54 55 38
55 56 17
56 57 17
57 58 35
58 59 9
59 60 46
60 61 46
61 62 24
62 63 24
63 64 26
64 65 1
65 66 45
66 67 13
67 68 7
68 69 50
69 70 2
70 71 1
71 72 38
72 73 20
73 74 1
74 75 15
75 76 47
76 77 5
77 78 12
78 79 12
79 80 12
80 81 50
81 82 4
82 83 35
83 84 39
84 85 45
85 86 9
86 87 10
87 88 46
88 89 45
89 90 8
90 91 15
91 92 20
92 93 40
93 94 10
94 95 6
95 96 5
96 97 4
97 98 8
98 99 45
99 100 3
100 101 27
101 102 49
102 103 34
103 104 19
104 105 16
105 106 44
106 107 28
107 108 9
108 109 28
109 110 10
110 111 4
111 112 47
112 113 35
113 114 24
114 115 15
115 116 37
116 117 13
117 118 45
118 119 21
119 120 27
120 121 14
121 122 34
122 123 42
123 124 1
124 125 38
125 126 39
126 127 40
127 128 42
128 129 3
129 130 34
130 131 15
131 132 23
132 133 24
133 134 33
134 135 2
135 136 45
136 137 41
137 138 2
138 139 35
139 140 30
140 141 48
141 142 23
142 143 35
143 144 5
144 145 46
145 146 12
146 147 4
147 148 40
148 149 1
149 150 48
150 151 2
151 152 21
152 153 22
153 154 38
154 155 43
155 156 7
156 157 43
157 158 11
158 159 22
159 160 28
160 161 38
161 162 30
162 163 25
163 164 22
164 165 35
165 166 26
166 167 39
167 168 30
168 169 14
169 170 44
170 171 47
171 172 5
172 173 32
173 174 2
174 175 12
175 176 7
176 177 4
177 178 50
178 179 24
179 180 13
180 181 29
181 182 23
182 183 3
183 184 36
184 185 42
185 186 27
186 187 19
187 188 5
188 189 30
189 190 27
190 191 17
191 192 23
192 193 20
193 194 36
194 195 21
195 196 17
196 197 8
197 198 8
198 199 37
199 200 36
200 201 19
201 202 42
202 203 17
203 204 21
204 205 29
205 206 6
206 207 11
207 208 29
208 209 28
209 210 20
210 211 14
211 212 12
212 213 29
213 214 41
214 215 50
215 216 33
216 217 5
217 218 14
218 219 5
219 220 27
220 221 36
221 222 29
222 223 45
223 224 29
224 225 46
225 226 3
226 227 44
227 228 5
228 229 3
229 230 7
230 231 6
231 232 21
232 233 27
233 234 6
234 235 38
235 236 31
236 237 49
237 238 32
238 239 11
239 240 19
240 241 37
241 242 8
242 243 40
243 244 41
244 245 46
245 246 32
246 247 24
247 248 19
248 249 44
249 250 37
250 251 6
251 252 2
252 253 23
253 254 34
254 255 33
255 256 6
256 257 23
257 258 5
258 259 17
259 260 35
260 261 22
261 262 6
262 263 41
263 264 48
264 265 33
265 266 43
266 267 25
267 268 19
268 269 26
269 270 12
270 271 2
271 272 34
272 273 2
273 274 18
274 275 18
275 276 16
276 277 25
277 278 23
278 279 21
279 280 20
280 281 22
281 282 1
282 283 18
283 284 8
284 285 26
285 286 42
286 287 28
287 288 21
288 289 32
289 290 45
290 291 1
291 292 49
292 293 29
293 294 9
294 295 42
295 296 25
296 297 42
297 298 15
298 299 40
299 300 29
300 301 12
301 302 6
302 303 46
303 304 8
304 305 39
305 306 39
306 307 6
307 308 16
308 309 35
309 310 31
310 311 18
311 312 32
312 313 34
313 314 42
314 315 1
315 316 42
316 317 48
317 318 19
318 319 44
319 320 6
320 321 43
321 322 19
322 323 41
323 324 44
324 325 25
325 326 36
326 327 30
327 328 41
328 329 37
329 330 6
330 331 17
331 332 35
332 333 8
333 334 8
334 335 8
335 336 47
336 337 3
337 338 47
338 339 21
339 340 24
340 341 3
341 342 27
342 343 42
343 344 50
344 345 6
345 346 29
346 347 6
347 348 22
348 349 50
349 350 5
350 351 33
351 352 35
352 353 44
353 354 10
354 355 31
355 356 36
356 357 36
357 358 41
358 359 38
359 360 25
360 361 20
361 362 25
362 363 30
363 364 24
364 365 13
365 366 48
366 367 23
367 368 43
368 369 42
369 370 44
370 371 10
371 372 1
372 373 1
373 374 1
371 374 20
212 294 26
222 317 50
82 156 22
68 81 6
336 352 8
276 369 6
157 174 31
176 223 20
215 300 46
71 130 48
189 243 26
116 210 29
5 93 36
288 371 49
88 181 18
64 125 49
2 12 21
320 326 46
53 128 15
259 274 47
141 169 48
170 219 14
107 163 1
5 79 27
38 54 36
185 231 23
40 92 42
241 267 41
67 104 31
97 171 50
169 209 32
242 254 46
164 197 42
28 95 45
222 308 23
151 180 9
122 141 4
90 178 7
350 367 20
112 181 4
80 90 28
250 270 23
359 371 4
18 89 3
209 287 41
248 297 13
254 324 32
250 313 16
169 189 6
77 127 11
316 339 17
332 371 10
119 218 26
354 371 50
250 311 6
353 357 37
145 243 38
67 93 38
277 292 40
327 371 45
56 128 36
142 211 9
234 291 43
23 32 45
361 371 15
1 98 12
256 289 7
112 161 15
189 202 38
109 153 45
148 160 28
129 145 1
182 259 21
106 144 28
245 254 29
4 104 46
84 89 1
77 166 46
156 191 12
363 368 15
12 80 27
122 153 13
167 234 7
231 286 45
304 371 8
68 110 42
149 198 41
58 83 50
107 168 45
222 321 43
259 270 16
104 114 22
145 219 10
364 371 13
99 178 12
312 336 40
363 371 10
35 104 16
104 134 48
38 127 20
69 73 43
89 167 34
58 91 41
108 187 50
208 292 48
339 371 24
131 160 11
303 337 16
369 371 46
231 251 1
85 114 20
172 260 35
290 323 50
272 282 28
72 96 30
351 371 34
182 273 12
269 365 47
52 101 47
271 352 7
31 68 10
353 370 42
22 119 21
26 102 7
321 371 23
8 23 48
158 246 43
67 147 35
20 66 5
287 323 34
84 103 40
235 249 31
10 57 5
196 264 7
194 274 37
253 301 30
150 181 42
127 192 22
249 289 31
284 342 15
185 280 38
182 233 15
169 182 33
121 134 20
156 206 10
177 273 24
195 252 37
346 371 44
81 138 33
224 227 6
234 274 7
57 135 27
146 213 39
162 212 37
320 371 42
366 371 35
347 371 1
218 272 32
27 76 5
65 106 37
262 285 24
335 371 46
323 371 11
275 346 14
321 347 17
154 232 32
54 82 33
305 344 20
178 206 16
357 371 36
76 91 34
262 295 10
277 370 6
239 290 29
293 330 15
43 60 10
228 306 34
117 152 2
48 89 1
73 108 33
232 332 50
93 121 26
101 139 47
7 89 28
245 278 10
283 297 7
133 220 19
186 262 7
268 286 15
305 340 6
40 48 50
268 333 22
72 89 24
204 211 10
85 147 42
35 127 9
22 25 22
132 147 41
125 156 34
26 92 30
192 252 23
111 128 45
186 286 21
293 360 13
248 338 17
36 130 33
11 50 6
53 135 37
199 252 28
43 89 22
310 344 40
343 371 37
206 294 26
266 273 43
262 326 22
107 145 12
72 75 5
247 271 27
61 75 47
40 82 15
60 130 33
216 304 32
233 258 4
159 206 6
94 150 48
121 164 35
278 293 31
88 108 40
177 233 12
365 371 33
83 173 10
349 371 15
193 267 48
83 114 40
155 250 36
242 278 9
1 80 8
256 343 27
224 282 46
42 135 11
156 167 11
12 31 36
273 355 36
245 259 8
313 371 17
201 292 1
225 282 32
189 214 26
144 207 11
36 99 5
186 225 38
330 371 9
44 50 1
206 232 22
18 68 37
313 342 12
130 187 47
200 263 21
196 208 34
281 347 31
59 159 29
69 80 25
226 278 17
7 103 1
169 227 11
159 205 13
21 104 4
65 114 12
353 371 9
274 288 32
280 288 33
334 371 23
240 275 47
111 208 16
156 166 14
138 192 35
240 331 25
243 314 15
268 301 6
27 93 39
277 369 32
294 371 14
205 210 3
56 90 23
149 248 24
196 246 48
192 257 5
297 336 3
97 132 21
234 333 14
332 357 15
2 32 39
17 116 33
50 88 34
312 371 17
142 209 36
128 159 32
314 338 1
138 183 22
208 283 47
143 193 1
97 168 21
352 371 44
309 348 26
300 336 19
117 156 22
14 77 3
262 361 11
134 154 47
227 269 5
350 356 50
265 279 32
14 39 19
312 328 7
312 333 49
311 371 36
289 361 19
318 329 3
325 371 50
137 194 33
280 350 22
98 182 24
293 344 43
202 205 40
117 175 42
310 315 46
285 370 24
191 220 42
352 358 29
293 371 45
177 179 32
227 296 1
307 364 41
180 230 28
234 237 6
242 300 5
18 26 36
223 273 32
287 294 26
179 231 13
199 257 22
307 350 15
306 371 23
152 249 12
9 57 30
262 311 38
285 333 34
52 146 34
171 208 44
206 303 3
330 354 25
78 142 41
219 254 11
192 247 28
308 349 24
331 371 8
56 85 21
154 207 29
273 295 29
171 199 38
288 302 5
219 285 35
104 172 21
103 169 37
68 161 21
135 159 3
96 166 17
117 141 17
260 295 17
355 371 45
340 371 12
133 208 48
244 284 15
145 208 49
215 280 7
118 177 43
220 287 32
202 207 25
214 313 27
222 278 47
144 210 41
130 183 16
177 203 23
185 242 37
325 331 4
274 280 17
114 189 50
41 69 8
20 110 50
109 206 24
14 80 35
284 329 21
39 118 18
263 331 44
65 150 26
0 64 47
262 291 8
176 258 45
65 128 32
205 239 29
205 283 46
324 367 31
256 274 25
130 210 46
2 86 44
198 298 35
103 188 34
158 177 14
196 282 48
282 333 1
140 207 49
102 133 20
45 77 12
102 130 47
91 162 29
270 327 15
224 310 8
212 219 39
152 209 18
66 123 1
64 68 13
130 219 44
142 181 36
80 95 37
245 268 18
337 371 33
221 276 20
192 213 8
211 226 25
181 209 30
261 303 13
52 150 44
342 371 48
147 239 5
324 357 23
292 338 20
47 125 26
298 371 44
222 281 36
304 344 21
74 131 32
11 16 27
12 52 1
30 34 8
143 211 34
290 371 50
241 333 11
135 171 39
107 128 45
126 142 42
171 261 12
13 41 49
141 235 20
232 238 45
128 223 18
144 216 28
3 93 23
101 201 2
244 286 15
271 360 15
