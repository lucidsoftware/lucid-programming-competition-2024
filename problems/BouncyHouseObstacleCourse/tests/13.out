168
0
1
80
90
178
206
303
304
371
374
